### PR TITLE
write docs, fix tests

### PR DESCRIPTION
**Pull request by morethanreal**
@cdata PTAL cc @notwaldorf

### CODE REVIEW

File: paper-dialog-scrollable.html [hunk 1]
```diff
@@ -13,12 +13,24 @@
...
+      </paper-dialog-scrollable>
+      <div class="buttons">
+        <paper-button>OK</paper-button>
+      </div>
+    </paper-dialog-impl>
+
+It shows a top divider after scrolling if it is not the first child in its parent container,
+indicating there is more content above. It shows a bottom divider if it is scrollable and it is not
+the last child in its parent container, indicating there is more content below. The bottom divider
+is hidden if it is scrolled to the bottom.
```
Comment (cdata) [inline paper-dialog-scrollable.html:33]: Sweet :100:

File: paper-dialog-scrollable.html [hunk 2]
```diff
@@ -89,34 +116,22 @@
...
-      }
-      if (this.scrollTarget.scrollTop + this.scrollTarget.offsetHeight < this.scrollTarget.scrollHeight) {
-        this.classList.remove('scrolled-to-bottom');
-      } else {
-        this.classList.add('scrolled-to-bottom');
-      }
+      this.toggleClass('is-scrolled', this.scrollTarget.scrollTop > 0);
+      this.toggleClass('can-scroll', this.scrollTarget.offsetHeight < this.scrollTarget.scrollHeight);
+      this.toggleClass('scrolled-to-bottom',
+        this.scrollTarget.scrollTop + this.scrollTarget.offsetHeight >= this.scrollTarget.scrollHeight);
```
Comment (cdata) [inline paper-dialog-scrollable.html:134]: Thanks, this method is a lot easier to read without all of the conditional statements :eyeglasses:

File: test/paper-dialog-scrollable.html [hunk 3]
```diff
@@ -86,63 +86,81 @@
 
     <script>
 
+      function runAfterScroll(node, scrollTop, callback) {
```
Comment (cdata) [inline test/paper-dialog-scrollable.html:89]: Is this general enough to go in test-helpers?

**Comment by cdata** [comment]
Woot, this is awesome :+1: LGTM

File: test/paper-dialog-scrollable.html (hunk 3 above)
Comment (morethanreal) [inline test/paper-dialog-scrollable.html:89]: Sure, I'll make a note to do it: https://github.com/PolymerElements/paper-dialog-scrollable/issues/7

**Pull request merged**